### DISCUSSION TITLE
Issue with Job Display and Total Count in Slurm-Web UI

### DISCUSSION THREAD

**Discussion by jakubn551** [rackslab/Slurm-web#410] [Q&A] [answered]
Hi,

I have Slurm with accounting enabled via `slurmdbd` and have successfully installed Slurm-Web. However, I am encountering an issue where, after 5 minutes, no jobs are displayed in the UI, and the total job count is reset to 0.

The `sacct` command is able to display past jobs without any issues. I assume this might be a configuration problem or a misunderstanding on my part, but I can't figure out from the documentation what needs to be adjusted.

Slurm version: 24.05.4 - build from source 
Slurm-web version: 3.2.0-1.deb12

Could you please assist in troubleshooting this issue?

**Comment by rezib** [comment, marked as answer]
Hello @jakubn551, this is actually the expected behavior for Slurm-web, at least for now. It displays the job queue, as reported by Slurm controller, similarly to `squeue` command, and completed jobs are only visible a few minutes.

The feature you expect is in the roadmap with #325. You will be able to explore completed jobs reported by SlurmDBD accounting service, similarly to `sacct` command.
